mp4parse_capi/src/lib.rs

```diff
@@ -114,16 +115,30 @@ pub struct mp4parse_track_info {
 }
 
 #[repr(C)]
+#[derive(Default, Debug)]
+pub struct mp4parser_indice {
```

Comment:
Please use `mp4parse_indice` for consistency with other structs. "mp4parser_noun" is more grammatical, but we can ignore that treating "mp4parse" as a prefix instead of an adjective. :)